mesa/space.py

```diff
@@ -507,6 +507,25 @@ def position_agent(
             coords = (x, y)
             self.place_agent(agent, coords)
 
+    def swap_pos(self, agent_a: Agent, agent_b: Agent) -> None:
+        """Swap agents positions"""
+        agents_no_pos = []
+        if (pos_a := agent_a.pos) is None:
+            agents_no_pos.append("agent_a")
+        if (pos_b := agent_b.pos) is None:
+            agents_no_pos.append("agent_b")
+        if agents_no_pos:
+            raise Exception(f"{', '.join(agents_no_pos)} - not on the grid")
```

Comment:
When debugging, I would have wanted to know the agent ID so that I can inspect it.

Reply:
[04f9e91](https://github.com/projectmesa/mesa/pull/1474/commits/04f9e91122a14c93f7a982e7c2d5b72f60b0a54a) - Show agents unique_id with no positions